.circleci/config.yml

```diff
@@ -70,6 +80,120 @@ jobs:
           name: Push Docker image to DockerHub
           command: docker push openmf/ph-ee-connector-channel:latest
 
+      - run: |
+          kubectl config use-context arn:aws:eks:$REGION:$ACCOUNT_ID:cluster/sit
```

Comment:
For bulk, ams mifos, channel and ops app, the restart logic is already added. You can skip this